buildSrc/assets/themes/railgun/mikoto/dark/mikoto.dark.github.definition.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": "01e61500-862c-41cd-b653-48d2ef7b1882",
+  "overrides": {},
+  "laf": {},
+  "syntax": {},
+  "colors": {}
```

Comment:
One of the things with Nadeshiko's & Mikoto's themes, is that their Accent Color, doesn't play well with the foregroundColor (hard to read the font in the buttons). 

If there is another foreground color in their theme that works I'll use that (like Ryuko's dark selection foreground), other wise I just use the base background, because that is always usable. I might come up with a dedicated foreground color to contrast with the accent color in the future for themes like that, if I want more color.

See https://github.com/doki-theme/doki-theme-github/blob/dc6f52ebc076bf27ce103d81367fc773c57e34b1/buildSrc/assets/themes/railgun/mikoto/dark/mikoto.dark.github.definition.json#L7